account_asset_management_stock_lot/models/account_asset.py

```diff
@@ -0,0 +1,30 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, exceptions, fields, models
+
+
+class AccountAsset(models.Model):
+    _inherit = "account.asset"
+
+    product_id = fields.Many2one(
+        "product.product",
+        string="Product",
+        compute="_compute_product_id",
+        store=True,
+    )
+    stock_lot_id = fields.Many2one("stock.lot", string="Lot")
```

Comment:
```suggestion
    stock_lot_id = fields.Many2one("stock.lot", string="Lot/Serial Number")
```
Maybe we could update the name to follow the Odoo way? https://github.com/odoo/odoo/blob/17.0/addons/stock/models/stock_move_line.py#L49
WDYT? 😄

Reply:
What is the Odoo way ? `lot_id` ?
I am not sure, because we are on an `account` model.
You would not name a many2one toward `stock.move` `move_id` on `account.asset` for instance, but you would name it `stock_move_id`, right ?

Reply:
Hi @florian-dacosta, I was only mentioning the `string` attribute to be changed, see in the suggestion. For the field name, I guess they are all fine already 👍🏿

Reply:
Ok, I'll rename the string